src/main/channels/audio/GenerateAudiosTranscriptionChannel.ts

```
@@ -6,33 +6,63 @@ import { instanceToPlain } from 'class-transformer';
       return;
     }
 
-    const { blobPath } = request.params;
+    try {
+      const transcription = await transcribeAudio({ blobPath, audioId });
+      log.info(`[${this.getName()}] Transcription successful`);
+      log.debug(`[${this.getName()}] Transcription result:`, transcription);
 
-    const transcription = await transcribeAudio(blobPath);
-    log.debug('transcription ' + transcription);
+      const safeContent =
```

ESLint

[error] 'safeContent' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)